[FIX] fieldservice

osi-scampbell opened the pull request.

max3903 commented on fieldservice/models/fsm_team.py:
```diff
@@ -59,6 +59,8 @@ def _compute_order_need_schedule_count(self):
         'fsm.order', 'team_id', string='Orders',
         domain=[('stage_id.is_closed', '=', False)])
+    sequence = fields.Integer('Sequence', default=1,
+                              help="Used to order stages. Lower is better.")
```
to order team...

max3903 commented on fieldservice/models/fsm_person.py:
```diff
@@ -30,6 +30,33 @@ class FSMPerson(models.Model):
+                                    'form')]
+                action['res_id'] = vendor_bills.id
+            return action
+
```
This should be moved in fieldservice_account and should not depend on purchase.

max3903 commented on fieldservice/views/fsm_person.xml:
```diff
@@ -43,6 +43,15 @@
+                                groups="base.group_user">
+                            <field name="bill_count" widget="statinfo" string="Vendor Bills"/>
+                        </button>
+                    </div>
```
To be moved to fieldservice_account.

max3903 committed:
fix

# Conflicts:
#	fieldservice/models/fsm_team.py
#	fieldservice/views/fsm_person.xml
